WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -102,9 +106,18 @@ private extension CartView {
 }
 
 #if DEBUG
+import Combine
 #Preview {
-    CartView(viewModel: PointOfSaleDashboardViewModel(itemProvider: POSItemProviderPreview(),
-                                                      cardPresentPaymentService: CardPresentPaymentPreviewService(),
-                                                      orderService: POSOrderPreviewService()))
+    // TODO:
+    // Simplify this by mocking `CartViewModel`
+    // https://github.com/woocommerce/woocommerce-ios/issues/13207
+    let orderStageSubject = PassthroughSubject<PointOfSaleDashboardViewModel.OrderStage, Never>()
+    let orderStagePublisher = orderStageSubject.eraseToAnyPublisher()
```

Comment:
In SwiftUI preview, we could also pass a constant value like passing `Just< PointOfSaleDashboardViewModel.OrderStage >(.building)` to `CartViewModel(orderStage:)`. We can potentially have two previews, one for each order stage case.

Would like to see what you think about this change, I will merge the PR without updating this 🙂

Reply:
Thanks for merging, appreciated! 🙇 

> In SwiftUI preview, we could also pass a constant value like passing Just< PointOfSaleDashboardViewModel.OrderStage >(.building) to CartViewModel(orderStage:). We can potentially have two previews, one for each order stage case.

That sounds good, I've updated https://github.com/woocommerce/woocommerce-ios/issues/13207 for reference